src/brevitas_examples/llm/llm_quant/prepare_for_quantize.py

```diff
@@ -21,3 +22,18 @@ def replace_mha_with_quantizable_layers(model, dtype):
     for rewriter in rewriters:
         model = rewriter.apply(model)
     return model
+
+
+@torch.no_grad()
+def add_zero_bias_to_linear(model: torch.nn.Module) -> torch.nn.Module:
```

Comment:
Is this here for loading checkpoint + bias correction?
We have a context manager for that now (load_quant_model in graph/calibrate)

Reply:
Ah no it's for accelerate compatibility, nevermind.

Reply:
No, this is to make bias correction work with accelerate properly. Can this also be handled by the context manager you mentioned?